ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,4 +4,8 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes: []
+  m_Scenes:
```

Comment:
Also these files should not have been checked in.
Assets/HoloToolkit/Input/Models/Cursor/cursor_hand_ready.fbm/templates.meta  
Assets/HoloToolkit/Input/Models/Cursor/cursor_ready.fbm/templates.meta